docs/content/en/docs/yaml-crd-ref/app.md

```diff
@@ -4,18 +4,16 @@ description: Define all workloads and checks associated with an application
 weight: 10
 ---
 
-A `KeptnApp` resource lists all the [workloads](https://kubernetes.io/docs/concepts/workloads/)
-that constitute a logical application.
+A `KeptnApp` is a Custom Resource Definition (CRD) that serves as a blueprint for defining and managing all the essential components, including [workloads](https://kubernetes.io/docs/concepts/workloads/) that are  necessary to operate our application within a Kubernetes cluster. This definition outlines the various pieces that make up our application, ensuring their proper orchestration and coordination.
 It contains information about:
 
 - All [workloads](https://kubernetes.io/docs/concepts/workloads/) and checks
   that are associated with a Keptn application
-- A list of tasks and evaluations to be executed
-  pre- and post-deployment.
-- Tasks referenced by `KeptnApp` are defined in a
-  [KeptnTaskDefinition](taskdefinition.md) resource.
-  `KeptnApp` identifies the task by the value of the `metadata.name` field
-  and does not need to understand what runner is used to define the task.
+- A list of tasks and evaluations to be can be executed such as 
```

Comment:
Extra words here...  Why do you say, "can be executed"?  If they are listed here, they will be executed.  "such as" also makes no sense -- all tasks defined in a KeptnApp are defined for either pre-deployment or post-deployment execution -- there are no other options.